server/state/state.test.js

```diff
@@ -1,46 +1,46 @@
-const smartContracts = require('./')
+const state = require('./')
```

Comment:
why the rename? smartContracts seemed more descriptive and state will eventually handle more than just a list of smart contracts

Reply:
I like naming it whatever we're calling the file myself. Makes it clear what we're testing.

Reply:
ok just in the future we'll have to change it all back because state.add() will be ambiguous what we're adding – we'd probably have two separate files at this point but I'm not sure. for me this rename makes it less clear but that's just personal style I think

Reply:
I think you both are right - since state is just smart contracts right now, why don't we rename the file smartContracts and then keep the more specific wordage?

Reply:
Word, maybe `smartContractState` then?